addons/web_editor/static/tests/test_utils.js

```diff
@@ -291,43 +291,34 @@ function wysiwygData(data) {
  *
  * @param {object} params
  */
-async function createWysiwyg(params) {
+async function createWysiwyg() {
     patch();
-    params.data = wysiwygData(params.data);
 
-    var parent = new Widget();
-    await testUtils.mock.addMockEnvironment(parent, params);
+    var params = {data: wysiwygData({})};
+    await testUtils.mock.addMockEnvironment(this, params);
 
-    var wysiwygOptions = _.extend({}, params.wysiwygOptions, {
+    var wysiwygOptions = _.extend(this._getWysiwygOptions(), params.wysiwygOptions, {
         recordInfo: {
             context: {},
             res_model: 'module.test',
             res_id: 1,
         },
         useOnlyTestUnbreakable: params.useOnlyTestUnbreakable,
     });
+    console.log('wysiwygOptions : ', wysiwygOptions);
+    this.wysiwyg = new WysiwygTest(this, wysiwygOptions);
+    this.wysiwyg._parentToDestroyForTest = this;
 
-    var wysiwyg = new WysiwygTest(parent, wysiwygOptions);
-    wysiwyg._parentToDestroyForTest = parent;
-
-    var $textarea = $('<textarea/>');
-    if (wysiwygOptions.value) {
-        $textarea.val(wysiwygOptions.value);
-    }
-    var selector = params.debug ? 'body' : '#qunit-fixture';
-    $textarea.prependTo($(selector));
-    if (params.debug) {
-        $('body').addClass('debug');
-    }
-    return wysiwyg.attachTo($textarea).then(function () {
+    var self = this
+    return this.wysiwyg.attachTo(this).then(function () {
```

Comment:
Don't need to change but just so you know: ```var self = this``` is the old way of referencing `this`.
Since we use the fat arrow function in javascript your could have changed the `function () {` into `() => {`.

Reply:
:+1: